src/Symfony/Component/Console/Tests/Helper/SymfonyQuestionHelperTest.php

```diff
@@ -79,7 +79,9 @@ public function testAskReturnsNullIfValidatorAllowsIt()
         $questionHelper = new SymfonyQuestionHelper();
         $questionHelper->setInputStream($this->getInputStream("\n"));
         $question = new Question('What is your favorite superhero?');
-        $question->setValidator(function ($value) { return $value; });
+        $question->setValidator(function ($value) {
+            return $value;
+        });
```

Comment:
should be reverted. The new PHP-CS-Fixer is not configured the right way, sorry about that.